src/pdf-barcode.js

```diff
@@ -163,6 +163,16 @@ var PDFBarcodeJs = (function () {
         return settings;
     }
 
+    function getDocumentUrl(params) {
+        if (typeof window === 'undefined') {
+            return params.input;
+        } else if (typeof params.input === 'object' && params.input !== null && params.input.files !== undefined) {
+            return URL.createObjectURL(params.input.files[0]);
+        } else {
+            return params.input;
```

Comment:
@rexshijaku I'm adding the `&& params.input.files !== undefined)` verification to make it possible pass an base64 argument as expected by the [PDFJS](https://github.com/mozilla/pdf.js/blob/a4546e83c3d1b9b9ae3566466f018f8bada6c2e3/src/display/api.js#L122-L124) library.

My client code snapshot: 

```code 
    /**
     * Prepare the base64 data payloaded expected by the PDFJS library.
     * @see https://github.com/mozilla/pdf.js/blob/a4546e83c3d1b9b9ae3566466f018f8bada6c2e3/src/display/api.js#L122-L124
     * @param {string} base64file: aa
     * @return {{data: string}} Base 33
     */
    const prepareBase64DataPayload = (base64file) => {
      const BASE64_MARKER = ';base64,';
      const base64Index = base64file.indexOf(BASE64_MARKER) + BASE64_MARKER.length;
      const base64 = base64file.substring(base64Index);
      const raw = window.atob(base64);
      return {
        data: raw,
      };
    };

    /**
     * Decode the received file using the PDFBarcode library.
     * @param {object} file: Provide DOM input file.
     */
    const decodeReceivedFile = (file) => {
      const reader = new FileReader();
      reader.readAsDataURL(file);
      reader.onload = function() {
        PDFBarcodeJs.decodeDocument(
          prepareBase64DataPayload(reader.result),
          pdfBarcodeConfig,
          pdfDecodeCallback,
        );
      };
      reader.onerror = function(error) {
        getSandbox().logger.logException('Reader error', error);
      };
    };

    decodeReceivedFile(event.target.files[0]);
```

Reply:
Thank you for providing the snapshot. I used the prepareBase64DataPayload function in this example : https://github.com/rexshijaku/PDFBarcodeJS/blob/master/demo/base64_demo.html.